docs/maintainers/standalone.md

```diff
@@ -0,0 +1,58 @@
+# play-frontend-govuk: Standalone usage
+
+This library is intended to be use as a transitive dependency via [play-frontend-hmrc](https://github.com/hmrc/play-frontend-hmrc) 
+which handles SASS compilation, routing, etc. However, if you are unable to use that dependency, and wish to use 
+`play-frontend-govuk` directly, please follow the steps below.
+
+## Adding to a project directly 
+
+1.  Add [Twirl](https://github.com/hmrc/play-frontend-govuk/releases) library in the App dependencies.
+    ```sbt
+    //build.sbt for Play 2.6
+    libraryDependencies += "uk.gov.hmrc" %% "play-frontend-govuk" % "x.y.z-play-26"
+    //or Play 2.7
+    libraryDependencies += "uk.gov.hmrc" %% "play-frontend-govuk" % "x.y.z-play-27"
+    ```
+
+1.  Add SASS assets to app/assets/stylesheets in application.scss to inherit / extend GovUk style assets / elements, e.g.:
+    ```
+    $govuk-assets-path: "/<your-project-context-root>/assets/lib/govuk-frontend/govuk/assets/";
+    
+    @import "lib/govuk-frontend/govuk/all";
+    
+    .app-reference-number {
```

Comment:
I think we should delete this `app-reference-number`. Teams have been adding it, thinking it's required! Maybe just put in a comment \\ Add your custom styles go here